runtime/doc/builtin.txt

```diff
@@ -4261,6 +4262,23 @@ getreginfo([{regname}])					*getreginfo()*
 		Can also be used as a |method|: >
 			GetRegname()->getreginfo()
 
+getregion({pos1}, {pos2})				*getregion()*
+		Returns the list of strings from {pos1} to {pos2}.
+		For possible values of {pos1} and {pos2} see |line()|.
```

Comment:
This function uses the line and column number from the specified position.  The line() function returns only the line number.  I think that should be mentioned here.   The supplied position argument specifies the starting and ending line number and column number.

Reply:
Updated.